pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -2352,15 +1948,174 @@ func TestAssignFlavors(t *testing.T) {
 					ReclaimablePods: tc.wlReclaimablePods,
 				},
 			})
-			if tc.clusterQueue.FlavorFungibility.WhenCanBorrow == "" {
-				tc.clusterQueue.FlavorFungibility.WhenCanBorrow = kueue.Borrow
+			if tc.clusterQueue.Spec.FlavorFungibility.WhenCanBorrow == "" {
+				tc.clusterQueue.Spec.FlavorFungibility.WhenCanBorrow = defaultFlavorFungibility.WhenCanBorrow
+			}
+
+			if tc.clusterQueue.Spec.FlavorFungibility.WhenCanPreempt == "" {
+				tc.clusterQueue.Spec.FlavorFungibility.WhenCanPreempt = defaultFlavorFungibility.WhenCanPreempt
+			}
```

Comment:
do we still need this? I think this is handled by the cache logic now.

Reply:
removed